### PR TITLE
Feature: Mobeli-Menu

### PR DESCRIPTION
**Pull request by ZikaZaki**
# Personal Portfolio: Adding Mobile-Menu

## In this pull-request we did the followings:
- [x] Used basic JavaScript syntax.
- [x] Used JavaScript to manipulate DOM elements.
- [x] Used JavaScript events.
- [x] Added the hamburger button, so when the user clicks (or taps) it the mobile menu appears.
- [x] Added the close (X) button, so when the user clicks (or taps) it the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

### CODE REVIEW

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+function toggleMenu(menu){
```
Comment (Munsa1) [inline main.js:1]: - [ ] When the user clicks on the links in the menu, they should be taken to the selected part of the site. For example, selecting the `About` link should take the user to the about section. This is part of the requirements.
image
- Kindly fix this

File: main.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    menu.classList.toggle('open');
```
Comment (Munsa1) [inline main.js:2]: - [ ] Also there is an overflow on your mobile menu page  which is causing items to not be properly aligned. Kindly fix this.

Yor design on load
image

Figma design
image

**Review by Munsa1 — COMMENTED** [review]
Hi @GraceAbike, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 2 above)
Comment (GraceAbike) [reply main.js:2]: Sorry I don't know what you want me to change 

image

File: main.js (hunk 1 above)
Comment (GraceAbike) [reply main.js:1]: But this project was Approved
image

Comment (GraceAbike) [reply main.js:1]: Can you please check the project very well, because it Okey for the mobile 

image
ok OK on my browser

**Review by Munsa1 — COMMENTED** [review]
Hi @ZikaZaki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] Fix the links in your mobile menu, the about link should take the user to the about section of the page when clicked and the mobile menu should disappear, that is a requirement.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**